src/main/java/com/invoiced/entity/AbstractEntity.java

```diff
@@ -334,7 +334,7 @@ public T deleteWithResponse() throws EntityException {
     return entities;
   }
 
-  public EntityList<T> list(String nextURL, HashMap<String, Object> queryParms)
```

Comment:
Aw, I was partial to my query parms.

Reply:
Haha, it can be how we wish I suppose but IDEA complained about it as a typo xD